module/documents/activity/utility.mjs

```diff
@@ -42,7 +42,7 @@ export default class UtilityActivity extends ActivityMixin(UtilityActivityData)
       icon: '<i class="fa-solid fa-dice" inert></i>',
       dataset: {
         action: "rollFormula",
-        visibility: this.roll.visible ? "all" : "creator"
+        visibility: this.roll.visible ? "all" : undefined
```

Comment:
Could be blank as well, no?

```suggestion
        visibility: this.roll.visible ? "all" : ""
```

Reply:
Well, a blank string will results in `data-visibility=""` being added, while this will result in nothing being added (`dnd5e-dataset` doesn't check for falsy values because we might want a blank string to be added).